src/pages/developers/tutorials/swap-any.mdx

```diff
@@ -270,7 +270,7 @@ Once your environment is set up, compile the contract and deploy it to localnet
 using the following command:
 
 ```
-yarn deploy-any:localnet
+npx hardhat deploy --name SwapToAnyToken --network localhost --gateway 0x5FC8d32690cc91D4c39d9d3abcBD16989F875707 --uniswap-router 0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0
```

Comment:
Can we use the same syntax here as in the other commands that were updated below? I believe it works better in terms of UX, this one forces the user to scroll horizontally, which is not a great experience.

i.e.:
```bash
npx hardhat swap-from-evm \
  --network localhost \
  --receiver 0xc351628EB244ec633d5f21fBD6621e1a683B1181 \
  --amount 1 \
  --target 0x05BA149A7bd6dC1F937fA9046A9e05C05f3b18b0 \
  --recipient 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 \
  --withdraw true
```